docs/bokeh/source/docs/gallery.json

```diff
@@ -468,55 +468,55 @@
       "name": "movies",
       "url": "https://demo.bokeh.org/movies",
       "alt": "Thumbnail featuring an interactive query tool for a set of IMDB data. Tool features a default graph of the Tomato Meter (x-axis) against the number of reviews (y-axis). Graph can be refined using multiple variables including cast names, director name, number of Oscar wins, year released, end year released, genre, dollar at the box office, and more.",
-      "desc": "An interactive query tool for a set of IMDB data.(<a target='_blank' href='https://github.com/bokeh/bokeh/tree/branch-3.0/examples/server/app/movies'>source code</a>)</em>"
+      "desc": "An interactive query tool for a set of IMDB data.(<a target='_blank' href='https://github.com/bokeh/bokeh/blob/main/examples/server/app/movies'>source code</a>)</em>"
```

Comment:
We don't actually reliably update the `main` branch on any sort of schedule at all, so this risks having a link that is out of date. Another option would be  to use `HEAD`, e.g. 

https://github.com/bokeh/bokeh/tree/HEAD/examples/server/app/movies 

which will point to the tip of the current development branch. Unfortunately that has the opposite problem of potentially being too new. 

I guess, of those two, I'd prefer `HEAD` but perhaps the link needs a caveat about "latest version" or something. 

Realistically the only proper solution is to have a way to properly version the links, but that will require some development on the sphinx extension.